core/features/org.fusesource.ide.core.feature/feature.xml

```diff
@@ -222,4 +222,11 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
+         id="org.fusesource.ide.camel.model.service.impl.v2233"
```

Comment:
2.23.3? is that correct? thought u switched to 2.24.1?

Reply:
a specific plugin is created for the previously embedded version.

Reply:
ah, forget my comment. Now I see how it works. Never mind.